test/test_segmentation.cpp

```diff
@@ -329,4 +329,208 @@ TEST_F(SegmentationTest, ForwardedExprsAreReplicated) {
   testValidate(fec.fusion(), out_tensors, {in_tensor}, __LINE__, __FILE__);
 }
 
+TEST_F(SegmentationTest, ForceFp16Simple) {
+  EnableOptionsGuard opt_guard;
+  EnableOptionsGuard::getCurOptions().set(EnableOption::IoToLowerPrecision);
```

Comment:
wondering if there is style preference here? static method goes through class scope rather than `opt_guard.getCurOptions()`?

Reply:
> static method goes through class scope rather than opt_guard.getCurOptions()?

That seems to be the right thing to do. Otherwise, readers may mistaken it as an instance method.